vendor/ember-suave/test-loader.js

```diff
@@ -1,7 +1,12 @@
 /* globals jQuery, QUnit */
 
 jQuery(document).ready(function () {
-  var TestLoaderModule = require('ember-cli/test-loader');
+  var testLoaderModulePath = 'ember-cli-test-loader/test-support/index';
+  if (!requirejs.entries[testLoaderModulePath]) {
```

Comment:
Could you add `requirejs` to the jshint globals on L1? That should take care of the failing test.